scheduler/pkg/server/control_plane.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright (c) 2024 Seldon Technologies Ltd.
+
+Use of this software is governed by
+(1) the license included in the LICENSE file or
+(2) if the license included in the LICENSE file is the Business Source License 1.1,
+the Change License after the Change Date as each is defined in accordance with the LICENSE file.
+*/
+
+package server
+
+import (
+	pb "github.com/seldonio/seldon-core/apis/go/v2/mlops/scheduler"
+)
+
+func (s *SchedulerServer) SubscribeControlPlane(req *pb.ControlPlaneSubscriptionRequest, stream pb.Scheduler_SubscribeControlPlaneServer) error {
+	logger := s.logger.WithField("func", "SubscribeControlPlane")
+	logger.Infof("Received subscribe request from %s", req.GetSubscriberName())
+
+	// on reconnect we send the current state of the servers to the subscriber (controller) as we may have missed events
```

Comment:
Does this comment refer to the general process happening outside SubscribeControlPlane? (given that we just reply with an empty `ControlPlaneResponse`). Doesn't the scheduler wait for the status of the controller before starting to send back server status updates back?

Reply:
good catch I dont think this is a relevant comment, will remove.